server/services/auth_service.go

```diff
@@ -100,3 +102,29 @@ func GetUserFromSession(c *fiber.Ctx) (*models.User, error) {
 
 	return user, nil
 }
+func InsertUserToSession(c *fiber.Ctx, user *models.User) int {
+	sess, err := GetStore(c)
```

Comment:
Please use `GetUserFromSession` function instead

Reply:
How am I supposed to take the user out of the session if here I put it in the session.
user not supposed to be inside the session

Reply:
Sorry my bad